src/Extensions/Abstractions/Build/Microsoft.Omex.Extensions.Abstractions.props

```diff
@@ -0,0 +1,22 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Default properties for omex projects -->
+  <PropertyGroup>
+    <!-- Enforce nulability check and language version -->
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <!-- Warnings as errors -->
+    <ErrorReport>prompt</ErrorReport>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+    <!-- Enforce checks -->
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
+    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <!-- Define common properties -->
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <CurrentYear>$([System.DateTime]::Now.ToString(yyyy))</CurrentYear>
+    <Copyright>Copyright © $(CurrentYear)</Copyright>
```

Comment:
>CurrentYear [](start = 29, length = 11)

Copyright message should be `Copyright (c) Microsoft Corporation. All rights reserved.`

It doesn't require the year.

Reply:
Okay thank you, that I will remove all of it completely in case if somebody else will decide to use it